### ISSUE TITLE
Switch from ggplot2 to dRacula

### ISSUE THREAD

**Issue by jrdnbradford** [dracula/dracula-theme#967]
In reference to the merged PR: https://github.com/dracula/draculatheme.com/pull/273, I appears I don't have the access to my repo for making the following changes:
- Repo name: `dRacula`
- Description: `Dracula theme for R`
- Link: `draculatheme.com/R`

I can do this myself if given the rights, even temporarily. I also presume some work will need to be done in this repo to remove the old `ggplot2` submodule and add `dRacula`.

**Comment by luxonauta** [comment]
Hey 👋🏻 Jordan, I just updated your permissions to admin the repo; sorry for this! 😅

**Comment by jrdnbradford** [comment]
@luxonauta no worries! I added the necessary changes to my repo. https://draculatheme.com/R is still using the `INSTALL.md` from the [`SwiftUI` repo](https://draculatheme.com/swiftui) (I think some redirect thing was happening before I updated my repo name) but I presume this will update on the next build of the site when it pulls the right `INSTALL.md`. 

I still see a `ggplot2` submodule in this repo, but I'm not sure if this functionality is being used anymore. Let me know if there's anything I can do myself to help out on this. Thanks again!
